index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="style.css"/>
+	<link rel="preconnect" href="https://fonts.gstatic.com"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet"/>
+</head>
+<body class="background">
+	<header>
+		<nav>
+			<a href="#home">home.</a>
+			<a href="#projects">projects.</a>
+			<a href="#about">about.</a>
+			<a href="#contact">contact.</a>
+		</nav>
+	</header>
+	<main id="home">
+		<div class="top" id="fade">
+		<h1>maria</h1>
+		<h1>gusova.</h1>
+	</div>
+	<div class="bottom" id="fade">
+		<h2>software</h2>
+		<h2>developer.</h2>
+	</div>
```

Comment:
Duplicate ID here, suggest using a class instead as you can reuse them infinitely